oaknational/Oak-Web-Application
# Fix/lesq 1074/onboarding hubspot signup [LESQ-1074]

## Description

Music year: 1976

- add a default value (false) for newsletter sign up on role selection page

## Issue(s)

Fixes #
Unable to proceed past onboarding unless you check newsletter sign up

## How to test

1. Go to https://deploy-preview-2797--oak-web-application.netlify.thenational.academy/onboarding _*while signed in*_
2. Click on 'No' to go to role selection page
3. You should be able to select a role and proceed without checking the newsletter sign up checkbox
4. You should also be able to check the newsletter sign up and see that value reflected [in the hubspot form submission](https://app-eu1.hubspot.com/submissions/24947711/form/50399960-674f-4e23-ad66-367fbdbcdbfb/submissions/)

## Screenshots
<img width="1044" alt="Screenshot 2024-09-18 at 15 45 11" src="https://github.com/user-attachments/assets/6803d94e-47a1-45b2-aaae-204ffd7b4177">


## Checklist

- [ ] Can continue past onboarding without signing up to the newsletter
- [ ] Can still sign up to the newsletter
